tests/program.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
if you'd rename it to `program.phpx` or something, you can get rid of most of the workarounds ;)

Reply:
Seems a bit hacky, but useful indeed :grin: